### PR TITLE
[kunlunxin] Fix the loss equal to nan problem during mask-rcnn training

### PR DESCRIPTION
**Pull request by liujiabin20810**
During the training of Mask-RCNN on a single node with one XPU device, the loss occasionally reached nan  in certain steps.
Down-scaling the learning rate to 0.08 fixed the problem.

### CODE REVIEW

**Comment by liujiabin20810** [comment]
The training log  path is  `/home/kunlunxin/liujiabin01/FlagPerf/training/result/run20231222214340#` @ xx.xx.xx.30 node

rank0.out.log
``` 
epoch: 1 state.map_bbox:0.11977678622672543  state.map_bbox:0.1224725110115393
[PerfLog] {"event": "TRAIN_END", "metadata": {"file": "/home/kunlunxin/liujiabin01/FlagPerf/training/benchmarks/mask_rcnn/pytorch/run_pretraining.py", "lineno": 117, "time_ms": 1703307649885, "rank": -1}}
[PerfLog] {"event": "FINISHED", "value": {"e2e_time": 54961.364691495895, "converged": false, "train_time": 54961.19849872589, "train_no_eval_time": 52043.070743083954, "pure_training_computing_time": 50990.66525220871, "throughput(ips)_raw": 4.26722863413244, "throughput(ips)_no_eval": 4.506498111108618, "throughput(ips)_pure_compute": 4.5995085343555315, "final_map_bbox": 0.11977678622672543, "final_map_segm": 0.1224725110115393}, "metadata": {"file": "/home/kunlunxin/liujiabin01/FlagPerf/training/benchmarks/mask_rcnn/pytorch/run_pretraining.py", "lineno": 142, "time_ms": 1703307649885, "rank": -1}}
```

**Comment by liujiabin20810** [comment]
Here is 2x8 MaskRcnn training log: `/home/kunlunxin/liujiabin01/FlagPerf/training/result/run20231222175353/` @ .30 node

using node 2 and node 30:

```
[PerfLog] {"event": "FINISHED", "value": {"e2e_time": 7122.876718044281, "converged": false, "train_time": 7122.554446220398, "train_no_eval_time": 6440.2885365486145, "pure_training_computing_time": 6347.602469205856, "throughput(ips)_raw": 49.39211102649872, "throughput(ips)_no_eval": 54.6245712445254, "throughput(ips)_pure_compute": 55.42218525918703, "final_map_bbox": 0.24331819666973659, "final_map_segm": 0.2337527634453022}, "metadata": {"file": "/home/kunlunxin/liujiabin01/FlagPerf/training/benchmarks/mask_rcnn/pytorch/run_pretraining.py", "lineno": 142, "time_ms": 1703246053990, "rank": -1}}
```

using epoch = 2

**Comment by yuzhou03** [comment]
maskrcnn 1x1 修改之前跑到900+ steps，出现loss NAN的报错，训练终止。
lr降为0.08之后，跑了2个epoch. 共计14658x2=29316 steps，训练正常结束。

**Review by yuzhou03 — APPROVED** [review]

**Review by shh2000 — APPROVED** [review]
